pkgs/applications/backup/nis-one-click-backup/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, cmake
+, ninja
+, qt6
+, wrapQtAppsHook
+, extra-cmake-modules
+}:
+
+stdenv.mkDerivation rec {
+  pname = "one-click-backup";
+  version = "1.2.2";
+
+  src = fetchFromGitLab {
+    owner = "dev-nis";
+    repo = "nis-one-click-backup-qt";
+    rev = "9a1ca839ca511d55716ffc8898e5d9ec9449ef18"; #version;
+    hash = "sha256-JbgmfKEpKRj/ph8dkW0IRulmAhk6Dwv/1Z4j3hUgPXk=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    wrapQtAppsHook
+    extra-cmake-modules
+  ];
+
+   buildInputs = [
+    qt6.full
+    qt6.qtbase
+  ];
+
+  meta = with lib; {
+    description = "Simple Program to backup folders to an external location by copying them";
+    homepage = "https://gitlab.com/dev-nis/nis-one-click-backup-qt";
+    changelog = "https://gitlab.com/dev-nis/nis-one-click-backup-qt/-/blob/${src.rev}/CHANGELOG.md";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ ];
+    mainProgram = "one-click-backup";
+    platforms = platforms.all;
+  };
+}
```

(to match `finalAttrs` suggestion above)

```suggestion
})
```